src/main/java/appland/startup/AppLandStartupActivity.java

```diff
@@ -17,6 +19,10 @@ public void runActivity(@NotNull Project project) {
             AppMapApplicationSettingsService.getInstance().setFirstStart(false);
 
             openToolWindowAndQuickstart(project);
+
+            var telemetry = TelemetryService.getInstance();
+            telemetry.notifyTelemetryUsage(project);
```

Comment:
Please note that this notification is sometime only shown in the log (toolwindow "Notifications".) Only two or three notifications are shown at once and there are sometimes more than that at startup. If AppMap's notifications happens to be in the set of hidden notifications, it would only be listed in the "notifications" panel and not shown again.

Reply:
As long as it meets JetBrains standards for collecting telemetry (which it seems like it does?) that's fine by me.

Reply:
As far as I know only paid plugin have to show an opt-in notification. Other plugins don't have to follow this rule strictly, so the AppMap plugin is fine, AFAIK.

Reply:
We could add a setting to the UI settings of the AppMap plugin

Reply:
I'm not sure - this does appear to speak generally to any plugin on the marketplace:
https://plugins.jetbrains.com/legal/approval-guidelines

Reply:
Oh, you're right, my statement was wrong.

We could
- make this opt-in instead of opt-out
- show the notification again at startup until the user either accepted or rejected the telemetry
- add a setting to the AppMap settings UI to allow to turn it off/on again later
What do you think?